### DISCUSSION TITLE
Easiest possible way to configure padding

### DISCUSSION THREAD

**Discussion by TheBlapse** [prabhuignoto/react-chrono#278] [Q&A] [answered]
Hey I wanted to configure padding to align the text vertically. Can anyone tell me the best possible way to do it if exist other than forking and customizing my own by editing the source code?

![image](https://user-images.githubusercontent.com/76247922/182795281-a861a6bc-e0d9-44d9-a7d6-24c471983823.png)

**Comment by prabhuignoto** [comment, marked as answer]
@TheBlapse there is no way to apply custom styles to the card at the moment, but we will add an option to do just that in the next release
